### DISCUSSION TITLE
AlwaysOnTop should get rid of OmniPresent

### DISCUSSION THREAD

**Discussion by minsoehan** [labwc/labwc#1391] [General]
I found setting a window always-on-top by ToggleAlwaysOnTop gets also omni-present action as ToggleOmniPresent action is used. I think it would be better ToggleAlwaysOnTop action is separated from omni-present function while we also have ToggleOmniPresent action.

**Comment by stefonarch** [comment]
There's an open issue https://github.com/labwc/labwc/issues/1151
